R/QC4a.R

```diff
@@ -21,6 +21,8 @@
 #' op de laatste meetjaar uit het databestand.
 #' @param plt_per_stof Voeg tijdserie plots per stof toe aan resultaat
 #' @param plt_per_reeks Voeg tijdserie plots per reeks toe aan resultaat
```

Comment:
plt_per_reeks heeft nu geen functie meer. In r.195 test je alleen op plt_put_reeks.